tests/tests-cli-spec-arg/wdio-with-all-passed.conf.js

```diff
@@ -0,0 +1,34 @@
+import url from 'node:url'
+import path from 'node:path'
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
+const outputDirPathAllPassed = path.resolve(__dirname, 'logs-AllPassedSpec')
+
+export const config =
+    {
+        capabilities: [{
+            browserName: 'chrome'
+        }],
+        logLevel: 'trace',
+        framework: 'mocha',
+
+        reporters: ['spec'],
+        services: ['webdriver-mock'],
+
+        mochaOpts: {
+            ui: 'bdd',
+            timeout: 15000
+        },
+        specs:
+        [
+            './mocha.test04.js',
+            [
+                './mocha.test02.js',
+                './mocha.test03.js'
+            ],
+            [
+                './mocha.test01.js'
+            ]
+        ],
```

Comment:
It seems like there is an issue resolving files properly when using the `Launcher` interface instead of the CLI, to workaround this problem, make the following change for paths:

```suggestion
        [
            path.resolve(__dirname, 'mocha.test04.js'),
            [
                path.resolve(__dirname, 'mocha.test02.js'),
                path.resolve(__dirname, 'mocha.test03.js')
            ],
            [
                path.resolve(__dirname, 'mocha.test01.js')
            ]
        ],
```